lucene/sandbox/src/java/org/apache/lucene/search/intervals/IntervalQuery.java

```diff
@@ -147,7 +153,7 @@ public IntervalWeight(Query query, float boost, ScoreMode scoreMode) {
 
     @Override
     public void extractTerms(Set<Term> terms) {
```

Comment:
should this method be removed too?

Reply:
Let's make removing extractTerms() completely a separate issue?

Reply:
I agree if we can get rid of it, lets do it. Not sure what purpose it serves still here. If we can't can we deprecate?

Reply:
+1 to deprecate and remove in a followup